test_pool/pcie/operating_system/test_os_p005.c

```diff
@@ -258,7 +258,7 @@ payload(void)
            updated_mem_base |= (mem_base_upper << P_MEM_BU_SHIFT);
            updated_mem_lim |= (mem_lim_upper << P_MEM_LU_SHIFT);
 
-           val_pcie_bar_mem_read(bdf, new_mem_lim + MEM_OFFSET_SMALL, &value);
+           value = val_mmio_read(new_mem_lim + MEM_OFFSET_SMALL);
```

Comment:
Please revert back these changes as this information will be taken from platform

Reply:
Agree